Scripts/Modules/LockModule/LockModule.cs

```diff
@@ -47,7 +47,13 @@ void Awake()
 
 		public bool IsLocked(GameObject go)
 		{
-			return go && (go.hideFlags & HideFlags.NotEditable) != 0;
+			if (!go)
+				return false;
+
+			if (go.transform.IsChildOf(transform)) // EditorVR objects (i.e. PlayerHead) are hidden but not locked
```

Comment:
This doesn't seem necessary. If the NotEditable flag is not set on EditorVR objects, then it shouldn't be locked.